randomstuff/client.py

```diff
@@ -762,13 +748,25 @@ async def get_waifu(self, plan:str, type:str) -> Waifu:
         if not plan in PLANS:
             raise InvalidPlanError("The plan provided is invalid.")
 
-        response = await self._session.get(f"{self._base_url}/{plan}/waifu", params={'type': type})
+        _check_coro(self)
+
+        if self.version == '4':
+            response = await self._session.get(f"{self._base_url}/{plan}/waifu", params={'type': type})
+        elif self.version == '5':
+            response = await self._session.get(f"{self._base_url}/premium/{plan}/waifu", params={'type': type})
 
         _check_status(response)
 
-        return Waifu(url=(await response.json())[0]['url'])
+        return Waifu(url=(await response.json()[0]['url']))
```

Comment:
This would raise error too. See the previous review. You have to put json() function inside paranthesis and then do the rest stuff outside paranthesis.